### PR TITLE
small refractor.

### PR DESCRIPTION
**Pull request by FuncGuy**
moved some of the code blocks to separate method.
used Objects.nonNull() for null check.

### CODE REVIEW

File: src/main/java/com/ecommerce/store/services/impl/ProductServiceImpl.java [hunk 1]
```diff
@@ -34,20 +36,24 @@ public Product updateProduct(ProductUpdationDto productDto) {
...
-                         existingProduct.setPrice(productDto.getPrice());
-                     }
-                     return repository.save(existingProduct);
+                checkAllFieldExists(productDto, existingProduct);
+                return repository.save(existingProduct);
                  }
             ).orElseThrow(() -> new ProductNotExistsException("Product doesn't exist in the system to update"));
     }
 
+    private void checkAllFieldExists(ProductUpdationDto productDto, Product existingProduct) {
```
Comment (prasadus92) [inline src/main/java/com/ecommerce/store/services/impl/ProductServiceImpl.java]: @FuncGuy I think the function could be better-called `modifyUpdatedFields()`

File: src/main/java/com/ecommerce/store/services/impl/OrderServiceImpl.java [hunk 2]
```diff
@@ -56,6 +56,12 @@ public Order placeOrder(OrderDto order) {
         return orderRepository.save(created);
     }
 
+    private void checkItemsAreEmpty(OrderDto order) {
```
Comment (prasadus92) [inline src/main/java/com/ecommerce/store/services/impl/OrderServiceImpl.java]: `checkIfItemsAreEmpty()`

File: src/main/java/com/ecommerce/store/services/impl/ProductServiceImpl.java (hunk 1 above)
Comment (FuncGuy) [reply src/main/java/com/ecommerce/store/services/impl/ProductServiceImpl.java]: done!

File: src/main/java/com/ecommerce/store/services/impl/OrderServiceImpl.java (hunk 2 above)
Comment (FuncGuy) [reply src/main/java/com/ecommerce/store/services/impl/OrderServiceImpl.java]: done!

**Pull request merged**